lib/utils/update.ts

```diff
@@ -25,26 +25,26 @@ const balenaUpdateInterval = 1000 * 60 * 60 * 24 * 1;
 
 let notifier: UpdateNotifier.UpdateNotifier;
 
-// `update-notifier` creates files to make the next
-// running time ask for updated, however this can lead
-// to ugly EPERM issues if those files are created as root.
-if (!isRoot()) {
-	notifier = UpdateNotifier({
-		pkg: packageJSON,
-		updateCheckInterval: balenaUpdateInterval,
-	});
-}
-
 export function notify() {
 	if (!notifier) {
-		return;
+		// `update-notifier` creates files to make the next
+		// running time ask for updated, however this can lead
+		// to ugly EPERM issues if those files are created as root.
+		if (isRoot()) {
+			return;
+		} else {
+			notifier = UpdateNotifier({
+				pkg: packageJSON,
+				updateCheckInterval: balenaUpdateInterval,
+			});
+		}
 	}
-
-	notifier.notify({ defer: false });
-
-	if (notifier.update != null) {
-		console.log(
-			'Notice that you might need administrator privileges depending on your setup\n',
-		);
+	const up = notifier.update;
+	if (up) {
+		notifier.notify({
+			defer: false,
+			message: `Update available ${up.current} → ${up.latest}\n
+https://github.com/balena-io/balena-cli/blob/master/INSTALL.md`,
```

Comment:
What URL do we want to use? -
1. https://github.com/balena-io/balena-cli/blob/master/INSTALL.md
2. https://www.balena.io/docs/reference/cli/
3. https://www.balena.io/docs/reference/cli/#installation
4. A tinyurl to any of the above, e.g. https://tinyurl.com/yymwhcaj

Option 3 would be good if it wasn't for the fact that the anchor link results in the **Installation** section title to get under the page header, leaving the **Getting Started** section prominent. Users would need some scrolling to figure it out, then click on a link to takes them to GitHub. All things considered, I think my preference is for option 1, without a tinyurl link. The CLI is usually used on large enough screens to accommodate the long URL, and then it's straight to the point and clear that the URL will land on INSTALL.md on GitHub.

Reply:
Concur.